packages/components/atoms/f-image-tile/src/components/_tests/ImageTile.test.js

```diff
@@ -7,4 +7,148 @@ describe('ImageTile', () => {
         const wrapper = shallowMount(ImageTile, { propsData });
         expect(wrapper.exists()).toBe(true);
     });
+
+    describe('props :: ', () => {
+        describe('isLink :: ', () => {
+            const isLinkTrue = true;
+            const isLinkFalse = false;
+
+            it.each([
+                ['https://www.google.com', isLinkTrue],
+                ['#', isLinkFalse]
+            ])('should update `href` to %s when set to %s', (expectedValue, isLink) => {
+                // Arrange
+                const propsData = { isLink, href: 'https://www.google.com' };
+
+                // Act
+                const wrapper = shallowMount(ImageTile, {
+                    propsData
+                });
+
+                const link = wrapper.find('[data-test-id="image-tile-link"]');
+
+                // Assert
+                expect(link.attributes('href')).toBe(expectedValue);
+            });
+        });
+
+        describe('tileId :: ', () => {
+            it('should apply `tileId` to the input id', () => {
+                // Arrange
+                const propsData = { tileId: 'chicken' };
+
+                // Act
+                const wrapper = shallowMount(ImageTile, {
+                    propsData
+                });
+
+                const input = wrapper.find('[data-test-id="image-tile-input"]');
+
+                // Assert
+                expect(input.attributes('id')).toContain(propsData.tileId);
+            });
+
+            it('should apply `tileId` to the label for', () => {
+                // Arrange
+                const propsData = { tileId: 'chicken' };
+
+                // Act
+                const wrapper = shallowMount(ImageTile, {
+                    propsData
+                });
+
+                const label = wrapper.find('[data-test-id="image-tile-label"]');
+
+                // Assert
+                expect(label.attributes('for')).toContain(propsData.tileId);
+            });
+
+            describe('imgSrc :: ', () => {
+                it('should apply `tileId` to the label for', () => {
+                    // Arrange
+                    const propsData = { imgSrc: 'https://via.placeholder.com/150' };
+
+                    // Act
+                    const wrapper = shallowMount(ImageTile, {
+                        propsData
+                    });
+
+                    const image = wrapper.find('[data-test-id="image-tile-image"]');
+
+                    // Assert
+                    expect(image.attributes('src')).toContain(propsData.imgSrc);
+                });
+            });
+
+            describe('isSelected :: ', () => {
+                const isSelectedTrue = true;
+                const isSelectedFalse = false;
```

Comment:
NIT: I think the code is self-explanatory enough just passing in booleans to `it.each` and naming the `isSelected` parameter as you have. If it were me I'd probably skip adding the two variables above

Reply:
I really like the explicitness of the variables, however I see your point. I'm going to leave as is for now but I am open to potentially revisiting this when I add further tests 👍